Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "CoordinateTransformations"
 uuid = "150eb455-5306-5404-9cee-2592286d6298"
-version = "0.6.1"
+version = "0.6.2"
```

Comment:
Hm, this does change the type of `PolarFromCartesian` to have parameters which could potentially break some things? Practically I'm not sure it will though.

Reply:
Ah, that's a fair point. I have no qualms against bumping to `0.7.0`, although I suspect that will trigger a lot of downstream PRs. Your call!